oca_dependencies.txt

```diff
@@ -2,3 +2,4 @@ account-financial-reporting
 server-ux
 community-data-files
 partner-contact
+server-tools https://github.com/Tecnativa/server-tools.git 11.0-mig-base_view_inheritance_extension
```

Comment:
We need server-tools added here. But please do not refer to separate branch. I will buildout will initially fail as long as https://github.com/OCA/server-tools/pull/1407 is not merged. Make clear your migration is dependent on completion of that PR. Rationale: otherwise you would need to change oca_dependencies.txt again after that PR is merged. And what would the situation be for another module that depended on some merge request on server-tools?